src/components/sections/adopters/Adopters.data.tsx

```diff
@@ -0,0 +1,63 @@
+const AdopterList = [
+  {
+    reverse: true,
+    bgColor: "#000000",
```

Comment:
specify the colors in theme and put id here instead...you can use that id to take colors from the theme
eg: id: redhat
theme.colors.adopters[adopterList.id]

Reply:
Already fixed !